<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdX">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>300</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdY">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>390</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTurn">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Turn</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>20</width>
     <height>800</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>720</y>
     <width>89</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSoloB">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>490</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1(B) : CPU(W)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDuo">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>650</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1 : 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCpuNet">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>570</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CPU : Network</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMode">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSoloW">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>570</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1(W) : CPU(B)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
